internal/xds/bootstrap/bootstrap.go

```diff
@@ -165,14 +181,16 @@ func (a *Authority) Equal(other *Authority) bool {
 type ServerConfig struct {
 	serverURI      string
 	channelCreds   []ChannelCreds
+	callCreds      []CallCreds
 	serverFeatures []string
 
 	// As part of unmarshalling the JSON config into this struct, we ensure that
 	// the credentials config is valid by building an instance of the specified
 	// credentials and store it here for easy access.
-	selectedCreds    ChannelCreds
-	credsDialOption  grpc.DialOption
-	extraDialOptions []grpc.DialOption
+	selectedCreds     ChannelCreds
```

Comment:
I am considering renaming this (and the `SelectedCreds` method) to include `Channel` in their name, as it's more symmetric and clearer with the addition of CallCreds. WDYT?